concourse/pipeline.yml

```diff
@@ -49,42 +33,77 @@ resources:
     source:
       interval: 24h
 
-  - name: feature-tests-image
+  - name: tests-image
     type: docker-image
     icon: docker
     source:
       repository: ((readonly_private_ecr_repo_url))
-      tag: govuk-coronavirus-business-volunteer-form-feature-tests
+      tag: govuk-coronavirus-business-volunteer-tests-image
 
 jobs:
   - name: update-pipeline
     plan:
-      - get: govuk-coronavirus-business-volunteer-form
+      - get: git-master
         trigger: true
       - set_pipeline: govuk-corona-business-volunteer-form
-        file: govuk-coronavirus-business-volunteer-form/concourse/pipeline.yml
+        file: git-master/concourse/pipeline.yml
 
-  - name: build-feature-tests-image
+  - name: build-tests-image
     serial: true
+    build_logs_to_retain: 100
     plan:
       - get: every-day
         trigger: true
-      - get: govuk-coronavirus-business-volunteer-form-feature-tests
+      - get: git-master-gems-dockerfile
         trigger: true
-      - put: feature-tests-image
+      - put: tests-image
         params:
-          build: govuk-coronavirus-business-volunteer-form-feature-tests
-          dockerfile: govuk-coronavirus-business-volunteer-form-feature-tests/spec/Dockerfile
+          build: git-master-gems-dockerfile
+          dockerfile: git-master-gems-dockerfile/concourse/Dockerfile
+
+  - name: run-quality-checks
+    serial: true
+    plan:
+      - get: tests-image
+        passed:
+          - build-tests-image
+        trigger: true
+      - get: git-master
+        trigger: true
+      - task: run-tests-task
+        image: tests-image
+        config:
+          inputs:
+            - name: git-master
+          outputs:
+            - name: committer-details
+          platform: linux
+          run:
+            dir: git-master
+            path: bash
+            args:
+              - -c
+              - |
+                set -eu
+                service postgresql start
+                su - postgres -c "psql -c \"create role root with createdb login password 'password';\""
+                service redis-server start
+                export TEST_DATABASE_URL="postgres://root:password@localhost:5432/coronavirus_business_volunteer_form_test"
```

Comment:
Should reflect this stuff back on the find-support form.
I think the app thinks it's running in development over there.